src/cli/main.py

```
@@ -254,56 +643,139 @@ def fetch(ctx, date_from, date_to, data_spec, jv_option, db, duckdb_memory_limit
         # Connect to database
         with database:
-            # Ensure tables exist
+            # Ensure tables exist (both JRA and NAR if source=all)
             try:
-                create_all_tables(database)
+                create_all_tables(database)  # JRA tables
+                if data_source == DataSource.ALL:
+                    # Also create NAR tables
+                    from src.database.schema_nar import get_nar_schemas
+                    nar_schemas = get_nar_schemas()
+                    for table_name, schema_sql in nar_schemas.items():
```

Ruff (0.14.14)

[warning] Loop control variable `table_name` not used within loop body

Rename unused `table_name` to `_table_name`

(B007)